geotrellis/build.sbt

```diff
@@ -19,36 +16,32 @@ publishMavenStyle := true
 publishArtifact in Test := false
 pomIncludeRepository := { _ => false }
 
-resolvers += Resolver.bintrayRepo("azavea", "geotrellis")
-
-val gtVersion = "1.0.0-2077839"
-
-val geotrellis = Seq(
-  "com.azavea.geotrellis" %% "geotrellis-accumulo"  % gtVersion,
-  "com.azavea.geotrellis" %% "geotrellis-hbase"     % gtVersion,
-  "com.azavea.geotrellis" %% "geotrellis-cassandra" % gtVersion,
-  "com.azavea.geotrellis" %% "geotrellis-s3"        % gtVersion,
-  "com.azavea.geotrellis" %% "geotrellis-spark"     % gtVersion,
-  "com.azavea.geotrellis" %% "geotrellis-spark-etl" % gtVersion
+val gtVersion        = "1.0.0-SNAPSHOT"
```

Comment:
If https://github.com/locationtech/geotrellis/pull/1947 is included in some published version except snapshot than yes.